js/join.js

```diff
@@ -6,6 +6,26 @@ import {
   SELF_URL,
 } from './constants.js';
 
+const reloadFunc = () => {
+  let flowState = +window.localStorage.getItem('flowState');
+  flowState += 1;
+  const currentPage = document.getElementById('page' + flowState);
+  const inputElements = currentPage.getElementsByTagName('input');
```

Comment:
You are selecting all the input of particular page...Is I am right??

Reply:
Yes